controllers/partner.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const mysql   = require('anytv-node-mysql');
+const winston = require('winston');
+const util = require(__dirname + '/../helpers/util');
+
+
+exports.partner_status = (req, res, next) => {
+    function start () {
+        if (!req.params.id) {
+            return next('id is missing');
+        }
+
+        mysql.use('dashboard_db')
+            .query(
+                'SELECT linked FROM channels WHERE channel_id = ? LIMIT 1',
+                [req.params.id],
+                send_response
+            )
+            .end();
+    }
+
+    function send_response (err, data) {
+        if (err) {
+            winston.error(err.message);
+            return next(err);
+        }
+
+        if (!data.length) {
+            return res.warn(404, 'Channel not found');
```

Comment:
res.warn will send a string instead of an object {message: 'Channel not found'}. we can improve this by adding res.error() or something alike